code/main/task/taskNode.brs

```diff
@@ -20,6 +20,18 @@ function _adb_createTaskNode() as void
     end if
 end function
 
+function _adb_storeTaskNode(taskNode as object) as void
+    if GetGlobalAA()._adb_main_task_node = invalid
+        GetGlobalAA()._adb_main_task_node = taskNode
+    end if
+end function
+
+function _adb_setTaskNode(taskNode) as void
```

Comment:
Where is this function used?

Reply:
Oh, it's merged from the POC branch, I should remove it.